include/diagnostics/scaling_ratio.hpp

```diff
@@ -0,0 +1,97 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 
+
+// Contributed and/or modified by Iva Janković, as part of Google Summer of Code 2025 program.
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+#ifndef DIAGNOSTICS_SCALING_RATIO_HPP
+#define DIAGNOSTICS_SCALING_RATIO_HPP
+
+template< typename Polytope, typename MT, typename VT, typename StreamType >
+void scaling_ratio_test(    const Polytope&   P,
+                            const MT&         samples,
+                            const VT&      facet_id,
+                            double            tol,
+                            StreamType&       out,
```

Comment:
I would delete this input variable. This function shouldn't print anything. It should return the required result as output (i.e., two vectors, the x- and the y-axis, right?).